tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,15 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    assert items == []
+    assert len(items) == 0
```

Comment:
If we've already asserted that the list is an empty list, I would say we don't really need to check and see if it also has a length of 0. I would say that asserting an empty list is the more robust of the two!